src/bag/core.py

```diff
@@ -665,8 +665,84 @@ def generate_cell(self, specs: Dict[str, Any],
 
         return final_netlist
 
-    def extract_cell(self, lib_name: str, cell_name: str, extract_type: Optional[str], extract_corner: Optional[str],
-                     netlist: str) -> None:
+    def lvs_cell(self, lib_name: str, cell_name: str) -> None:
+        """Run layout-vs-schematic (LVS) on a cell. Exports netlist and GDS from OA views.
+        
+        Parameters
+        ----------
+        lib_name : str
+            Library name
+        cell_name : str
+            Cell name
+        
+        """
+        print('running LVS...')
+        lvs_passed, lvs_log = self.run_lvs(lib_name, cell_name, run_rcx=True)
```

Comment:
`run_rcx` should be False in this `lvs_cell()` method, since this is for LVS only. `extract_cell()` should be used for `run_rcx=True`.